dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -42,4 +42,7 @@ timeout=Cache eviction timeout
 timeout.desc=Amount of time after which an entry in the cache will be removed.
 
 allowBasicAuthLookup=Allow lookup by user ID and hashed password
-allowBasicAuthLookup.desc=Allow lookup by user ID and hashed password.
\ No newline at end of file
+allowBasicAuthLookup.desc=Allow lookup by user ID and hashed password.
+
+jCacheRef=JCache reference
+jCacheRef.desc=The JCache reference that will be used as the authentication cache. Use of JCache results in the initialSize, maxSize and timeout configuration attributes being ignored as the size and eviction are controlled by the configuration of the JCache implementation used.
```

Comment:
- Change "will be used" to "is used"
- Add comma after "maxSize" (nit)
- Change "JCache implementation used" to "JCache implementation"

Reply:
9/7/2021 update:  Messages are not complete. No need to review yet.